### ISSUE TITLE
How should text following <textPath> be positioned?

### ISSUE THREAD

**Issue by progers** [w3c/svgwg#104]
Where should ```after``` be positioned in the following scenario:
```<text>before<textPath xlink:href="#path">on path</textPath>after</text>```
(I've created an example showing this at http://jsbin.com/gugoje)

Currently, WebKit, Blink, and Edge position ```after``` above ```before``` in the y-axis, and after ```on path``` in the x-axis. Gecko appars to just reset the position of ```after``` to 0,0.

One approach is to position ```after``` as if ```on path``` was a tspan.

**Comment by fsoder** [comment]
To me it looks like Gecko reset the current text position (to 0,0) on the <textPath> (not at "after"; easily observable by replacing the textPath with a tspan with x=y=0). That seems to be closer to the last approach - albeit with a bug somewhere... (My guess would be an incorrect computation of the initial CTP.)

**Comment by nikosandronikos** [comment]
RESOLUTION: text following textPath text is positioned as if the x,y is at the end point of the equivalent path

https://www.w3.org/2016/04/22-svg-minutes.html#resolution07

Note the equivalent path hasn't been written up yet, but see https://www.w3.org/2016/04/22-svg-minutes.html#resolution02.

**Comment by BigBadaboom** [comment]
Sorry for the late comment...

Wouldn't it be most logical that the x,y of "after" would be whatever the current text position was after the last character rendered on the path (ie. the "h")?

**Comment by AmeliaBR** [comment]
@BigBadaboom That was the other possibility we discussed, and seems to match the current Edge/Blink behavior.  The Gecko behavior is definitely undesirable, and spec will be changed where it suggests that is correct.

The reason we resolved on using the end of the path, rather than the position of the last character, is that it would provide more precise author control, with less variability based on font, etc.

If other authors or implementers want to argue for keeping the Edge/Blink behavior, now would be the time.
